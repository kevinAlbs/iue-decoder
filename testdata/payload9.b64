CVkwKtDmKE4wmHqxJleVlkgQ6M6Fzuh68tGrCjKyAHq9M6tNsvO6BY8gnwWXlI/fHpDV/AjlPQA/bB4zz+GPulalwDYVuIKRecthdXAThkOVyIoJ1DnEqk2b3V7mZ00ZU8DIivKqTPtJWVfgCkUGufk/q0Y2zQ74jXvaHCSfJ2XzwHfmKCuEaKDqiPYGhMN1QqSmA/EJ9phTeASi8tLz4pq9dZx0Ad0xDUikxQWeZsr1oKn3RABKB0c3+SXie+tLp56KKkUvQQdv+iK+d77dFhRWgGXX4n53oyJrd8sinsc/BM7mdMxk/8B+0gQr2j0so7f2fqfnqz4szX1w7+ecYda2H1etE3gXODcTIrsu/p4V3ReKwkFgtF60z+ho2WW0F3PVKMgNieWNYuEYVRA8afm6b08YjJoHFGl43zL5Tm+eHCdj5fZ9eWdF8Ym1tV9W9ZPhtLYyFXKi/6q9nLSBJLxnQpiiTmvIuszMbMLBVN9xPMc7gLPw1KqXhHmLXla5o47nrDsfg9GwURGybTmGQ8GCyKrodD3hqwAyUd9ket+jwF6GOqATJEv839qHW06M6vtYJP1g7quCxAzHPlhC9bN3yixY2QWGkJCE9KiEA+wKq4tRkhKXvGCXHLwE+Xv6KJDD+dqbyArjgFZfU6b/o0NJVqsucNgfC5hoeKtK9ihoMvpCiY59J6idiwyYUhnONHJEaRQbyVempbAG4e4H5DbXGLWTLHyAJrgG3S9gXgT3AbdlU4Rq/XYX/ZIzBdkOt8d4T4UohPxvxLoKcO05RjTd3AhskhtQ6NPhvHp302egfwSAcUblJ5sYFdubs8109LRW8fr9GfU4tN0QSMudcxU6BgGldaSPHgzEqj9ne5MvGAA+62ZxNwBW4qBAyiUEut/J0fEm3sZ+kIJEyMo4KuG8s4zaBwDPRQ6BvzODmkfloiFXEQmeLPgaRQBdCw3gLmMlbRJ2mxQ4d6oW/REkqDZTxlmm9jnu/z5DrDgJc3kugHIib8BHe/MgUinestjj97jCwRLJw+iJ+Roo/HWL288nKGY/stokjHUY4BY0k1T77R+54kyeMPX9lL/jGbfoozJCDiSSxIBUbY6mDwoEtpmIyzrG/arT+cL/NmQW+CvgfQCQr+nk2+BdUmK1C8uqYJbBKmZHOYh5ugla0dDqP0ybLsiBKr2qUsG5nOq+/wdZe/K5Z7Xjv4vI2xCYdbTaIL6XfhM1cqV46B3tar4h4JVmNiglVuqC37h4gzhuA93/5ZB4e2E//Mnv5LKa9NCgV8/B2DLLt2FQRSfdQeiMUUjwd7VJTXAfhRiO3eGH171QBPGqdak9ek65WLvvHboyCYm/Wa42UTkIGq56kX6IBk/APgZCx76Kc5GdW4pT4dmUp9rwtq8CjmTA