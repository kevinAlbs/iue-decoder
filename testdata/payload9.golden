BlobSubtype=9
key_uuid=59302ad0e6284e30987ab12657959648
OriginalBsonType=16
InnerEncrypted=e8ce85cee87af2d1ab0a32b2007abd33ab4db2f3ba058f209f0597948fdf1e90d5fc08e53d003f6c1e33cfe18fba56a5c03615b8829179cb61757013864395c88a09d439c4aa4d9bdd5ee6674d1953c0c88af2aa4cfb495957e00a4506b9f93fab4636cd0ef88d7bda1c249f2765f3c077e6282b8468a0ea88f60684c37542a4a603f109f698537804a2f2d2f3e29abd759c7401dd310d48a4c5059e66caf5a0a9f744004a074737f925e27beb4ba79e8a2a452f41076ffa22be77bedd1614568065d7e27e77a3226b77cb229ec73f04cee674cc64ffc07ed2042bda3d2ca3b7f67ea7e7ab3e2ccd7d70efe79c61d6b61f57ad13781738371322bb2efe9e15dd178ac24160b45eb4cfe868d965b41773d528c80d89e58d62e11855103c69f9ba6f4f188c9a07146978df32f94e6f9e1c2763e5f67d796745f189b5b55f56f593e1b4b6321572a2ffaabd9cb48124bc674298a24e6bc8bacccc6cc2c154df713cc73b80b3f0d4aa9784798b5e56b9a38ee7ac3b1f83d1b05111b26d398643c182c8aae8743de1ab003251df647adfa3c05e863aa013244bfcdfda875b4e8ceafb5824fd60eeab82c40cc73e5842f5b377ca2c58d90586909084f4a88403ec0aab8b51921297bc60971cbc04f97bfa2890c3f9da9bc80ae380565f53a6ffa3434956ab2e70d81f0b986878ab4af6286832fa42898e7d27a89d8b0c985219ce34724469141bc957a6a5b006e1ee07e436d718b5932c7c8026b806dd2f605e04f701b76553846afd7617fd923305d90eb7c7784f852884fc6fc4ba0a70ed394634dddc086c921b50e8d3e1bc7a77d367a07f04807146e5279b1815db9bb3cd74f4b456f1fafd19f538b4dd1048cb9d73153a0601a575a48f1e0cc4aa3f677b932f18003eeb6671370056e2a040ca2504badfc9d1f126dec67e908244c8ca382ae1bcb38cda0700cf450e81bf33839a47e5a2215711099e2cf81a45005d0b0de02e63256d12769b143877aa16fd1124a83653c659a6f639eeff3e43ac380973792e8072226fc0477bf3205229deb2d8e3f7b8c2c112c9c3e889f91a28fc758bdbcf2728663fb2da248c7518e016349354fbed1fb9e24c9e30f5fd94bfe319b7e8a332420e2492c480546d8ea60f0a04b69988cb3ac6fdaad3f9c2ff366416f82be07d0090afe9e4dbe05d5262b50bcbaa6096c12a6647398879ba095ad1d0ea3f4c9b2ec8812abdaa52c1b99ceabeff07597bf2b967b5e3bf8bc8db109875b4da20be977e133572a578e81ded6abe21e0956636282556ea82dfb87883386e03ddffe590787b613ffcc9efe4b29af4d0a057cfc1d832cbb761504527dd41e88c5148f077b5494d701f85188edde187d7bd5004f1aa75a93d7a4eb958bbef1dba320989bf59ae365139081aae7a917e88064fc03e0642c7be8a73919d5b8a53e1d994a7daf0b6af028e64c0
